tests/unit/sparql/QueryBuilderSparqlGenerator.spec.ts

```diff
@@ -1,11 +1,15 @@
 import QueryBuilderSparqlGenerator from '@/sparql/QueryBuilderSparqlGenerator';
-import SparqlGenerator from '@/sparql/SparqlGenerator';
-import Sparqljs from '@/sparql/Sparqljs';
+import {
+	Generator as SparqlGenerator,
+	Parser as SparqlParser,
+	SelectQuery,
+} from "@/@types/sparqljs";
```

Comment:
just trying to import it from "sparqljs" didn't work in my localhost, I don't know why.

Reply:
Worked for me. Uploaded fixes!